packages/node/src/methods/state-channel/deposit/operation.ts

```diff
@@ -0,0 +1,148 @@
+import { StateChannel } from "@counterfactual/machine";
+import {
+  AssetType,
+  Node,
+  SolidityABIEncoderV2Struct
+} from "@counterfactual/types";
+import { AddressZero, One, Zero } from "ethers/constants";
+import { BigNumber } from "ethers/utils";
+
+import { RequestHandler } from "../../../request-handler";
+import { DepositConfirmationMessage, NODE_EVENTS } from "../../../types";
+import { getPeersAddressFromChannel } from "../../../utils";
+import { ERRORS } from "../../errors";
+
+export interface ETHBalanceRefundAppState extends SolidityABIEncoderV2Struct {
+  recipient: string;
+  multisig: string;
+  threshold: BigNumber;
+}
+
+export async function installBalanceRefundApp(
+  requestHandler: RequestHandler,
+  params: Node.DepositParams
+) {
+  const {
+    publicIdentifier,
+    instructionExecutor,
+    networkContext,
+    store
+  } = requestHandler;
+
+  const [peerAddress] = await getPeersAddressFromChannel(
+    publicIdentifier,
+    store,
+    params.multisigAddress
+  );
+  const initialState: ETHBalanceRefundAppState = {
+    recipient: publicIdentifier,
+    multisig: params.multisigAddress,
```

Comment:
dont use the params, use the official value from the DB (`stateChannel.multisigAddress`)